ostd/src/task/task/mod.rs

```diff
@@ -154,9 +154,7 @@ pub struct Task {
     link: LinkedListAtomicLink,
     cpu: AtomicCpuId,
     priority: Priority,
-    // TODO: add multiprocessor support
-    #[allow(dead_code)]
-    cpu_affinity: CpuSet,
+    pub cpu_affinity: CpuSet,
```

Comment:
Public fields are not acceptable.

Reply:
Public fields may be a good option to speak out our compromises. See the latest change for details